tools/topology/topology2/cavs/cavs-passthrough-hdmi.conf

```diff
@@ -50,151 +51,17 @@ IncludeByKey.NUM_DMICS {
 	"[1-4]"	"platform/intel/dmic-generic.conf"
 }
 
-Object.Dai {
-	HDA.4 {
-		name iDisp1
-		id 1
-		default_hw_conf_id 1
-		Object.Base.hw_config.HDA4 {}
-		direction playback
-	}
-	HDA.5 {
-		name iDisp2
-		id 2
-		default_hw_conf_id 2
-		Object.Base.hw_config.HDA5 {}
-		direction playback
-	}
-	HDA.6 {
-		name iDisp3
-		id 3
-		default_hw_conf_id 3
-		Object.Base.hw_config.HDA6 {}
-		direction playback
-	}
-}
-Object.Pipeline {
-	passthrough-playback.1 {
-		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.6.1'
-		}
-		Object.Widget.copier.1 {
-			stream_name 'Passthrough Playback 3'
-		}
-
-		format s16le
-		index 5
-	}
-	passthrough-be.1 {
-		direction	"playback"
-		index 6
-		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.6.1'
-		}
-
-		Object.Widget.copier.1 {
-			stream_name "iDisp1"
-			dai_type "HDA"
-			copier_type "HDA"
-		}
-		format s16le
-	}
-	passthrough-playback.2 {
-		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.8.1'
-		}
-		Object.Widget.copier.1 {
-			stream_name 'Passthrough Playback 4'
-		}
-
-		format s16le
-		index 7
-	}
-	passthrough-be.2 {
-		direction	"playback"
-		index 8
-		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.8.1'
-		}
-
-		Object.Widget.copier.1 {
-			stream_name 'iDisp2'
-			dai_type "HDA"
-			copier_type "HDA"
-		}
-		format s16le
-	}
-	passthrough-playback.3 {
-		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.10.1'
-		}
-		Object.Widget.copier.1 {
-			stream_name 'Passthrough Playback 5'
-		}
-
-		format s16le
-		index 9
-	}
-	passthrough-be.3 {
-		direction	"playback"
-		index 10
-		Object.Widget.pipeline.1 {
-			stream_name 'copier.HDA.10.1'
-		}
-
-		Object.Widget.copier.1 {
-			stream_name 'iDisp3'
-			dai_type "HDA"
-			copier_type "HDA"
-		}
-		format s16le
-	}
-}
-Object.PCM {
-	pcm.2 {
-		name HDMI1
-		id 3
-		Object.Base.fe_dai.HDMI1 {}
-		Object.PCM.pcm_caps.playback {
-			name 'Passthrough Playback 3'
-			formats 'S32_LE,S24_LE,S16_LE'
-		}
-		direction playback
-	}
-	pcm.3 {
-		name HDMI2
-		id 4
-		Object.Base.fe_dai.HDMI2 {}
-		Object.PCM.pcm_caps.playback {
-			name 'Passthrough Playback 4'
-			formats 'S32_LE,S24_LE,S16_LE'
-		}
-		direction playback
-	}
-	pcm.4 {
-		name HDMI3
-		id 5
-		Object.Base.fe_dai.HDMI3 {}
-		Object.PCM.pcm_caps.playback {
-			name 'Passthrough Playback 5'
-			formats 'S32_LE,S24_LE,S16_LE'
-		}
-		direction playback
-	}
+Define {
+	HDMI "true"
+	HDMI1_ID	1
+	HDMI2_ID	2
+	HDMI3_ID	3
+	HDMI4_ID	8
```

Comment:
1,2 ,3 ,8 ?

Reply:
> We need to define this 20 as a macro somewhere so that the other HDMI1 definitions can reuse it.

@ranj063 Could you help on it? Sorry, I have no idea how to create a macro on a .conf file

> 1,2 ,3 ,8 ?

@lgirdwood It is intentional, see https://github.com/thesofproject/linux/pull/3685.